### PR TITLE
Events tests

### PR DESCRIPTION
**Pull request by dandipietrantonio**
Wrote tests for Sunny's event view. Now it's just missing tests for the export event as CSV feature (which is currently broken) and Grafana.

### CODE REVIEW

File: mercury/tests/test_event.py [hunk 1]
```diff
@@ -223,3 +257,119 @@ def test_update_event(self):
...
+            reverse(self.event_activate_url, kwargs={"event_uuid": event.uuid})
+        )
+
+        self.assertEqual(302, response.status_code)
+
+        response = self.client.post(
+            reverse(self.event_deactivate_url, kwargs={"event_uuid": event.uuid})
+        )
+
+        self.assertEqual(302, response.status_code)
```
Comment (daisycrego) [inline mercury/tests/test_event.py:376]: similarly you could assert that the event is no longer the active event

Comment (dandipietrantonio) [reply mercury/tests/test_event.py:376]: updated, thanks!

File: mercury/tests/test_event.py [hunk 2]
```diff
@@ -223,3 +257,119 @@ def test_update_event(self):
...
+        self.assertEqual(200, response.status_code)
+
+    def test_activate_event(self):
+        event = self.create_venue_and_event(self.event_name)
+
+        response = self.client.post(
+            reverse(self.event_activate_url, kwargs={"event_uuid": event.uuid})
+        )
+
+        self.assertEqual(302, response.status_code)
```
[the remark this answers is not in the document]
Comment (dandipietrantonio) [reply mercury/tests/test_event.py:361]: updated, thanks!

File: mercury/tests/test_event.py [hunk 3]
```diff
@@ -223,3 +257,119 @@ def test_update_event(self):
...
+    def test_export_json(self):
+        event = self.create_venue_and_event(self.event_name)
+        sensor = self.create_sensor()
+        self.create_measurement(event, sensor)
+
+        response = self.client.post(
+            reverse(self.event_export_json_url, kwargs={"event_uuid": event.uuid})
+        )
+
+        self.assertEqual(200, response.status_code)
```
[the remark this answers is not in the document]
Comment (dandipietrantonio) [reply mercury/tests/test_event.py:346]: updated, thanks!

**Pull request merged**